pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -497,6 +450,66 @@ func clearStatus(tr *v1alpha1.TaskRun) {
 	tr.Status.PodName = ""
 }
 
+func getTaskrunAnnotations(pr *v1alpha1.PipelineRun) map[string]string {
```

Comment:
It looks like these are still covered by the mega-tests in taskrun_test/pipelinerun_test, but do you think it might be worthwhile to simplify those tests and add coverage here instead?

Reply:
Yeah, will add some!

Reply:
Done!